common/djangoapps/student/views.py

```diff
@@ -1754,6 +1755,9 @@ def create_account_with_params(request, params):
             }
         )
 
+    # Announce registration
+    REGISTER_USER.send(sender=None, user=user, profile=profile)
```

Comment:
Could you use the post-save signal (https://docs.djangoproject.com/en/1.8/ref/signals/#post-save) for either `User` or `UserProfile` to avoid having to create a new signal for registration? Or, alternately, I think it would make more sense for the `student` package to define the signal, and for the `email_marketing` package to import that signal and listen for it (rather than having `student` depend directly on `email_marketing`).